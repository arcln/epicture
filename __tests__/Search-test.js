import 'react-native';
import React from 'react';
import Search from '../screens/SearchScreen';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';

describe('Search snapshot', () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it('renders the screen', async () => {
    const tree = renderer.create(<Search />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
