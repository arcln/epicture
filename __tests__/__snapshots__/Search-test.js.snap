// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot renders the screen 1`] = `
<View>
  <View
    style={
      Object {
        "backgroundColor": "#fff",
        "paddingTop": 20,
      }
    }
  >
    <View
      style={
        Object {
          "alignContent": "center",
          "borderBottomColor": "#888",
          "borderBottomWidth": 0.2,
          "flexDirection": "row",
          "paddingHorizontal": 20,
          "paddingVertical": 10,
        }
      }
    >
      <Text />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Search Imgur"
        style={
          Object {
            "fontSize": 18,
            "marginLeft": 10,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "flex-end",
        "padding": 10,
      }
    }
  >
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "marginRight": 10,
            }
          }
        >
          <Text />
          <Text
            style={
              Object {
                "color": "#2f95dc",
              }
            }
          >
             
            Most recent
          </Text>
        </View>
      </View>
    </View>
    <View>
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <Text />
          <Text
            style={
              Object {
                "color": "#2f95dc",
              }
            }
          >
             
            All time
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "height": 1334,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#ededed",
          "flex": 1,
          "justifyContent": "flex-start",
        }
      }
    >
      <View />
      <View
        style={
          Object {
            "height": 1334,
            "paddingBottom": 50,
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        />
      </View>
    </View>
  </View>
</View>
`;
